app/views/questions/show.html.haml

```diff
@@ -1,92 +1,90 @@
-                      / Modal Body
-                      .modal-body
-                        = form_for(@response, url: responses_path) do |f|
-                          %textarea#your-response.form-control{:name => "response[text]", :rows => "10"}
-                          = f.hidden_field(:question_id, :value => @question.id)
-                          / Modal Footer
-                          .modal-footer
-                            %button.btn.btn-default{"data-dismiss" => "modal", :type => "button"}
-                              Cancel
-                            = f.submit "Respond", class: "btn btn-success"
-              .col-sm-2
-                %td= link_to "Edit", edit_question_path(@question), class: "btn btn-md btn-info"
-              .col-sm-1  
-                %td
-                  = form_for @question, url: question_path(@question.id), html: {:onsubmit => "return confirm('Are you sure you want to delete the question?');"}, method: :delete do |f| 
-                    = f.submit "Delete", class: "btn btn-sm btn-danger"
+%table
+  %tbody
+    %tr
+      %td
+        %button#vote_up.btn.btn-success.glyphicon.glyphicon-thumbs-up{:type => "button"}
+          = button_to '+', up_vote_question_path(@question), method: :post
+          = @question.up_votes.count
+      %td
+        %button#vote_down.btn.btn-danger.glyphicon.glyphicon-thumbs-down{:type => "button"}
+          = button_to '-', down_vote_question_path(@question), method: :post
+          = @question.down_votes.count
+      %td 	
+        %h1= @question.body
+%table
+  %tbody
+    %tr
+      %td  
+        = link_to "Edit", edit_question_path(@question), class: "btn btn-default"          
+      %td 
+        = form_for @question, url: question_path(@question.id), html: {:onsubmit => "return confirm('Are you sure you want to delete the question?');"}, method: :delete do |f| 
+          = f.submit 'Delete', class: "btn btn-default"             
+=form_for @question, url: question_path(@question.id) do |f|                            
+  =f.select(:flags, options_for_select([['Flag',''],['Duplicate', 'Duplicate'], ['Off Topic', 'Off Topic'],['Contains inappropriate langauge','Contains inappropriate langauge']]), {}, {:style => "max-width: 100px;"})      
+  =f.submit 'Submit', url: question_path(@question.id), class: 'btn btn-default'
 
-        .panel-footer{:color => "green"}
-          %h4
-            %strong Responses:
-        .container-fluid
-          .row
-            - if @question.tags != nil  
-              %p
-                =@question.tags
-            - if @responses.size == 0
-              %h3 No Responses
-            - else
-              - @responses.each do |response|
-                %tr
-                  %td
-                    = link_to response.text, response
-                  %td
-                    = response.created_at.strftime("%m-%d-%Y")
-                    %br/
-                    %br/
+%h3
+  = "User: #{@question.user_id}"
```

Comment:
get the user's name: `"#{@question.user.first_name} #{@question.user.last_name}"`